docs/mobile/events.md

```diff
@@ -8,12 +8,27 @@ events on mobile platforms. All mobile events MUST use a namespace of
 
 <!-- toc -->
 
+- [Application unresponsive UI](#application-unresponsive-ui)
```

Comment:
Is `unresponsive_ui` sufficient?

Reply:
Sorry, I'm not sure I'm following. Sufficient as in "the name is descriptive enough"? Or as in "this event alone is enough to represent the issue"?